libr/bp/bp.c

```diff
@@ -167,6 +167,10 @@ static RBreakpointItem *r_bp_add(RBreakpoint *bp, const ut8 *obytes, ut64 addr,
 	}
 	b = r_bp_item_new (bp);
 	b->addr = addr + bp->delta;
+	if (bp->baddr > addr) {
+		eprintf ("base addr should not be larger than the breakpoint address.\n");
+	}
+	b->orig_addr = addr - bp->baddr;
```

Comment:
this is more like a delta

Reply:
so i would find a better name for this. also it will be good to see some feedback from @ret2libc because this baddr thing is very related to the rbin refactoring